README.md

```diff
@@ -62,6 +62,13 @@ docker run -p 3000:3000 tdm:v1.0.0
 
 The frontend will be accessible at `http://localhost:3000/` and the swagger configuration at `http://localhost:3000/swagger-config`.
 
+## Development mode
+
+To use the dev mode add the `http://localhost:3000` prefix to the servers url in `tdm-be/src/swagger/swagger.js`.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏What about `tdm-be/config/development.json`?

It seems that you have to use `ngrok` to set `hosts.internal.host` (in https).
We should describe it here too.

Was `tdm-be/src/swagger/swagger-config.json` generated too? I don't remember.
Same question for `tdm-be/src/swagger/swagger.json`. They seem to depend from `tdm-be/src/swagger/swagger.js`, indeed.

Reply:
The `swagger-*.json` files are generated via the `update-front-api` files